src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -222,6 +222,12 @@ public void RepetiviteBuildUpdateSingleResource ()
 				var firstBuildTime = b.LastBuildTime;
 				Assert.IsTrue (b.Build (proj), "Second build was supposed to build without errors");
 				Assert.IsTrue (firstBuildTime > b.LastBuildTime, "Second build was supposed to be quicker than the first");
+				Assert.IsTrue (
+					b.Output.IsTargetSkipped ("_UpdateAndroidResgen"),
+					"The Target _UpdateAndroidResgen should have been skipped");
+				Assert.IsTrue (
+					b.Output.IsTargetSkipped ("_UpdateAndroidResgen"),
+					"The Target _UpdateAndroidResgen should have been skipped");
```

Comment:
Did this just get pasted twice?

Reply:
Is it also worth adding a check that `IncrementalClean` didn't delete the new `$(_AndroidResFlagFile)`?

Reply:
weird..